src/components/user-list-item.component.js

```diff
@@ -1,65 +1,72 @@
 import React, { Component } from 'react';
 import { connect } from 'react-redux';
-import {
-  StyleSheet,
-  View,
-  TouchableHighlight,
-  TouchableOpacity,
-  Text,
-  Image,
-} from 'react-native';
+import { View, TouchableHighlight } from 'react-native';
 import { Icon } from 'react-native-elements';
+import styled from 'styled-components/native';
 
 import { colors, fonts, normalize } from 'config';
 
-const styles = StyleSheet.create({
-  borderContainer: {
-    borderBottomColor: colors.greyLight,
-    borderBottomWidth: 1,
-  },
-  wrapper: {
-    padding: 10,
-    flexDirection: 'row',
-  },
-  userInfo: {
-    flex: 1,
-    flexDirection: 'row',
-    alignItems: 'center',
-  },
-  avatarContainer: {
-    backgroundColor: colors.greyLight,
-    borderRadius: 17,
-    width: 34,
-    height: 34,
-  },
-  avatar: {
-    borderRadius: 17,
-    width: 34,
-    height: 34,
-  },
-  titleSubtitleContainer: {
-    justifyContent: 'center',
-    flex: 1,
-  },
-  title: {
-    color: colors.black,
-    ...fonts.fontPrimary,
-    fontSize: normalize(14),
-    marginLeft: 10,
-  },
-  subtitle: {
-    color: colors.greyDark,
-    fontSize: normalize(10),
-    marginTop: 1,
-    ...fonts.fontPrimarySemiBold,
-    marginLeft: 10,
-  },
-  iconContainer: {
-    flex: 0.15,
-    alignItems: 'flex-end',
-    justifyContent: 'center',
-  },
-});
+const ViewBorderContainer = styled.View`
+  border-bottom-color: ${props => props.hasBorderBottom && colors.greyLight};
```

Comment:
I think this will evaluate to a boolean and not work as expected .. shouldn't you be providing two different return values (just like the line below this one?)

Reply:
@shmesa22 Right. It will cause a `invalid prop` error.

I am eager to merge styled-components PRs and your PR is very very close to be merged. So forgive me to fix it in your branch directly. Normally I shouldn't do that.